pages/collections/[colId]/[colItemId].js

```diff
@@ -82,16 +88,25 @@ export async function getStaticPaths() {
 }
 
 export async function getStaticProps({ params }) {
-  const items = getItems(params.colId);
-  const item = items[params.colItemId];
+  const items = getItems(params.colId),
+    item = items[params.colItemId],
+    itemIds = Object.keys(items),
+    currentIndex = itemIds.indexOf(params.colItemId),
+    nextItem = itemIds[currentIndex + 1],
```

Comment:
Add a check here to make sure there is a next item, to avoid running off the end of the array?  I think that's why your test is failing.

Reply:
I like that I did the test for the previous item but not the next 😆 . It's now updated so that first/last items will restart the item loop